frontend/public/components/service-instance.tsx

```diff
@@ -0,0 +1,123 @@
+          <dl className="co-m-pane__details">
+            <dt>External Name</dt>
+            {/* TODO:  add link */}
+            <dd>{si.spec.clusterServiceClassExternalName || si.spec.serviceClassExternalName}</dd>
+            <dt>Status</dt>
+            <dd><StatusWithIcon obj={si} /></dd>
+            <dt>Plan</dt>
+            <dd>{siPlan || '-'}</dd>
+          </dl>
+        </div>
+      </div>
+    </div>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Conditions" />
+      <Conditions conditions={si.status.conditions} />
+    </div>
+    {!_.isEmpty(si.spec.parametersFrom) && <ServiceCatalogParametersSecrets obj={si} /> }
+    {!_.isEmpty(siParameters) && <ServiceCatalogParameters parameters={siParameters} /> }
+  </React.Fragment>;
+};
+
+const pages = [navFactory.details(ServiceInstanceDetails), navFactory.editYaml(), navFactory.events(ResourceEventStream)];
+export const ServiceInstanceDetailsPage: React.SFC<ServiceInstanceDetailsPageProps> = props =>
+  <DetailsPage
+    {...props}
+    kind={ServiceInstancesReference}
+    menuActions={menuActions}
+    pages={pages} />;
+ServiceInstanceDetailsPage.displayName = 'ServiceInstanceDetailsPage';
+
+const ServiceInstancesHeader = props => <ListHeader>
+  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="col-md-2 col-sm-4 hidden-xs">Status</ColHead>
+  <ColHead {...props} className="col-md-2 hidden-sm hidden-xs" sortField="metadata.creationTimestamp">Created</ColHead>
+  <ColHead {...props} className="col-md-2 hidden-sm hidden-xs" sortField="spec.clusterServicePlanExternalName || spec.servicePlanExternalName">Plan</ColHead>
+</ListHeader>;
+
+const ServiceInstancesRow: React.SFC<ServiceInstancesRowProps> = ({obj}) => <div className="row co-resource-list__item">
+  <div className="col-md-3 col-sm-4 col-xs-6 co-resource-link-wrapper">
+    <ResourceCog actions={menuActions} kind={ServiceInstancesReference} resource={obj} />
+    <ResourceLink kind={ServiceInstancesReference} name={obj.metadata.name} namespace={obj.metadata.namespace} title={obj.metadata.name} />
+  </div>
+  <div className="col-md-3 col-sm-4 col-xs-6 co-break-word">
+    <ResourceLink kind="Namespace" name={obj.metadata.namespace} title={obj.metadata.namespace} />
+  </div>
+  <div className="col-md-2 col-sm-4 hidden-xs co-break-word">
+    <StatusWithIcon obj={obj} />
+  </div>
+  <div className="col-md-2 hidden-sm hidden-xs co-break-word">
+    <Timestamp timestamp={obj.metadata.creationTimestamp} />
+  </div>
+  <div className="col-md-2 hidden-sm hidden-xs co-break-word">
+    {obj.spec.clusterServicePlanExternalName || obj.spec.servicePlanExternalName}
+  </div>
+</div>;
+
+const ServiceInstancesList: React.SFC = props => <List {...props} Header={ServiceInstancesHeader} Row={ServiceInstancesRow} />;
```

Comment:
In a follow on, we should look at adding list row filters for Ready / Pending / Failed

Reply:
Updated https://jira.coreos.com/browse/CONSOLE-702 with the new todo.